aiven/client/cli.py

```diff
@@ -5550,6 +5548,21 @@ def organization__list(self) -> None:
         ]
         self.print_response(organizations, json=self.args.json, table_layout=layout)
 
+    @arg.json
+    @arg.organization_id
```

Comment:
If we have no other organization commands yet, I think we should use
```suggestion
    @arg.positional_organization_id
```
as this is mandatory, and is part of the "context" of the CLI

`avn organization update <org_id> [-a abc] [-n new_name] [-e etc]`